src/bro_cli/cli.rs

```diff
@@ -30,10 +35,18 @@ pub fn process_cli(matches: clap::ArgMatches) {
     }
     if let Some(matches) = matches.subcommand_matches("run") {
         if matches.get_one::<String>("git_repo").is_some() {
-            println!(
-                "Running git repo {}, bro!",
-                matches.get_one::<String>("git_repo").unwrap()
-            );
+            let git_repo = String::from(matches.get_one::<String>("git_repo").unwrap());
```

Comment:
Doing ```if x.is_some() { x.unwrap() }``` is a common mistake (I think there's even a clippy lint about it), instead you can already match on the `Option` by using the `if let` matcher (like you did in line 36 with `subcommand_matches("run")` :
```rust
// DON'T DO THIS
if matches.get_one::<String>("git_repo").is_some() {
    // oh no, we didn't actually GET the git repo so we have to call `get_one` again, this is costly AND ugly
    let git_repo = String::from(matches.get_one::<String>("git_repo").unwrap());
}
// =========================================================================================================
// Instead, do this:
if let Some(git_repo) = matches.get_one::<String>("git_repo") { 
   // nice, it works the same, we only called 'get_one' once, and we already have 'git_repo'! win-win-win
   let git_repo = String::from(git_repo);
}
```
> [!TIP]
> **Q:** Why would `let git_repo = String::from(git_repo)` even compile? the new variable has the **exact same name** as the old one.. this feels wrong..
> **A:** It works thanks to a concept called [shadowing](https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing). The TL;DR is that rust allows you to declare a new variable with the same name as an already existing variable, because the old one becomes inaccessible anyway (you can't name it anymore) and therefore it can be safely destroyed.

Reply:
Found a ton of useful logic getting rid of unwraps.